src/main/java/org/mule/extensions/jms/internal/source/JmsListener.java

```diff
@@ -142,11 +142,12 @@
   private String inboundEncoding;
 
   /**
-   * The number of concurrent consumers that will be used to receive JMS Messages
+   * The number of concurrent consumers that will be used to receive JMS Messages. The default number of consumers is the
+   * available processors of the runtime environemnt.
```

Comment:
`The default number of consumers is the available number of cores of the runtime environment`
also typo on `environemnt` -> `environment`